rules/integrations/google_workspace/collection_google_drive_ownership_transferred_via_google_workspace.toml

```diff
@@ -76,7 +76,10 @@ The Google Workspace Fleet integration, Filebeat module, or similarly structured
 - See the following references for further information:
   - https://support.google.com/a/answer/7061566
   - https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-google_workspace.html"""
-references = ["https://support.google.com/a/answer/1247799?hl=en"]
+references = [
+    "https://support.google.com/a/answer/1247799?hl=en",
+    "https://www.elastic.co/security-labs/google-workspace-attack-surface-part-two",
```

Comment:
https://www.elastic.co/security-labs/google-workspace-attack-surface-part-one
https://www.elastic.co/security-labs/google-workspace-attack-surface-part-two

I'd just add these to each rule for Google Workspace then. They are referenced in docs.